db/util/random.go

```diff
@@ -0,0 +1,39 @@
+package util
+
+import (
+	"math/rand"
+)
+
+// List of owners names for testing
+var nameList = []string{"John", "Jane", "Joe", "Jill", "Jack"}
```

Comment:
Generate a random length of string should be fine in this case. 
``` suggestion
const alphabet = "abcdefghijklmnopqrstuvwxyz"
```

Reply:
What's the rule to generate the owner's name if we are going to use `GetRandomStringWithLength()`

Reply:
I would say just use `GetRandomStringWithLength` as the method for generating a random user name. I should clear that up and I updated in ticket.

Reply:
Sounds good, left comment on the ticket.